### ISSUE TITLE
508-defect-3: Supplemental Claims | Fix focus on private authorization page

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#54459]
### Point of contact

Josh Kim

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

During an accessibility audit, a few things that were found and need to be addressed. This ticket combines a few:
- HIPAA authorization page: when unchecked, the focus should move to the `h3` inside the alert, and not the alert itself
- <del>Private evidence limitation page: text area is overflowing on small screens</del>

### Reproduction steps

1.
2.
3.

### Proposed solution or next steps

- Move alert focus to the H3
<del>- Limit textarea width on small screens</del>

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [X] Screenreader
- [ ] Keyboard
- [X] Focus
- [ ] Headings
- [ ] Color
- [X] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by joshkimux** [comment]
The iOS bug on text area overflowing doesn't effect the experience/usability, so we've removed it from this ticket

**Comment by joshkimux** [comment]
@saderagsdale will do before EoD! 😀

**Comment by joshkimux** [comment]
<img width="557" alt="Screenshot 2023-03-08 at 4 02 02 PM" src="https://user-images.githubusercontent.com/14154792/223849048-474abe97-6a61-429f-8c34-35d04a31383d.png">

Confirming it's fixed 😄
